front-end/lambda-in/src/Components/DevInfoEdit/DevInfoEdit.js

```diff
@@ -0,0 +1,232 @@
+import React, { Component } from 'react'
```

Comment:
Move folder and files:
Currently, the DevInfoEdit component is in:

front-end/lambda-in/src/App.js
front-end is the wrong folder. Please move it to:

frontend/lambda-in/src/App.js